### PR TITLE
Bug 1590734 CNS required for rhgs3 images

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1590734

### CODE REVIEW

File: install/disconnected_install.adoc [hunk 1]
```diff
@@ -253,6 +253,11 @@ $ docker pull registry.access.redhat.com/rhgs3/rhgs-s3-server-rhel7:<tag>
 +
 [IMPORTANT]
 ====
+For Red Hat support, a CNS subscription is required for `rhgs3/` images.
```
Comment (kalexand-rh) [inline install/disconnected_install.adoc]: Are they supported only if you have the subscription, or do you need the subscription to obtain the images at all?

I'd spell out CNS.

**Review by kalexand-rh — COMMENTED** [review]
I have a question and a suggestion. :cat:

File: install/disconnected_install.adoc (hunk 1 above)
Comment (bmcelvee) [reply install/disconnected_install.adoc]: Thanks! They're only supported if you have the subscription.

**Pull request merged**